.github/workflows/macos_debug.yml

```diff
@@ -14,6 +14,9 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+    - uses: jwlawson/actions-setup-cmake@v1.9
+      with:
+        cmake-version: '3.18.x'
```

Comment:
Do we not get a new enough cmake through brew further down?

Reply:
I'll try this again, remember I had most of the Github actions failing but I don't know if I was waiting enough for the macos to run, I'll update here

Reply:
Yep this was indeed unecessary thanks!